plugins/woocommerce-blocks/assets/js/blocks/product-collection/utils.tsx

```diff
@@ -96,26 +96,34 @@ export function getDefaultValueOfInheritQueryFromTemplate() {
 }
 
 /**
- * Add Product Collection block to the parent array of the Core Pagination block.
+ * Add Product Collection block to the parent or ancestor array of the Core Pagination block.
  * This enhancement allows the Core Pagination block to be available for the Product Collection block.
  */
-export const addProductCollectionBlockToParentOfPaginationBlock = () => {
+export const addProductCollectionToQueryPaginationParentOrAncestor = () => {
 	if ( isWpVersion( '6.1', '>=' ) ) {
 		addFilter(
 			'blocks.registerBlockType',
 			'woocommerce/add-product-collection-block-to-parent-array-of-pagination-block',
 			( blockSettings: Block, blockName: string ) => {
-				if (
-					blockName !== coreQueryPaginationBlockName ||
-					! blockSettings?.parent
-				) {
+				if ( blockName !== coreQueryPaginationBlockName ) {
 					return blockSettings;
 				}
 
-				return {
-					...blockSettings,
-					parent: [ ...blockSettings.parent, blockJson.name ],
-				};
+				if ( blockSettings?.ancestor ) {
+					return {
+						...blockSettings,
+						ancestor: [ ...blockSettings.ancestor, blockJson.name ],
+					};
+				}
+
+				if ( blockSettings?.parent ) {
+					return {
+						...blockSettings,
+						ancestor: [ ...blockSettings.parent, blockJson.name ],
```

Comment:
```suggestion
						parent: [ ...blockSettings.parent, blockJson.name ],
```

I believe it should be parent instead of ancestor for older Wordpress versions. no? 🤔
I tested with Wordpress 6.4 and it doesn't seem to be working as expected. I believe this could be the problem.